src/reformat_libyuv.c

```diff
@@ -255,49 +222,37 @@ avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb)
     // Find the correct libyuv YuvConstants, based on range and CP/MC
     const struct YuvConstants * matrixYUV = NULL;
     const struct YuvConstants * matrixYVU = NULL;
-    if (image->yuvRange == AVIF_RANGE_FULL) {
+    if (image->yuvRange == AVIF_RANGE_LIMITED) {
```

Comment:
If I understand it correctly, all the remaining changes are just to test `image->yuvRange` or `image->yuvFormat` in the order of the enum values. I think it's not necessary to make this kind of change.

Also, to separate monochrome from non-monochrome YUV formats, it is cheaper to test `image->yuvFormat == AVIF_PIXEL_FORMAT_YUV400` (the only monochrome format) than testing three formats.